src/theVoice/TheVoice.tsx

```diff
@@ -1,21 +1,17 @@
 import React, { ReactNode } from 'react'
-import cc from 'classcat'
 
 import { StyledTheVoice } from './TheVoice.style'
 
 export type TheVoiceProps = Readonly<{
+  children: ReactNode
   id?: string
   className?: string
-  children: ReactNode
   isInverted?: boolean
+  as?: keyof JSX.IntrinsicElements | React.ComponentType<any> // from styled-components types
```

Comment:
Maybe can we only allow some tags like `h1 | h2 | h3` ?

Reply:
We're using a default styled-component feature here. We might render it as p, span or div, it doesn't matter IMHO.

Reply:
IMO if we're gonna add "as" to a component then it should behave like the other instances with that prop, so no restriction I guess.

Reply:
Ok. I'm more in favour of providing only what is needed right now and edit it for new usage latter. Smallest API first for an easy maintenance and robustness in the long term.

For example are we sure we want to allow `span` or `div` here? And even if we want to allow them, is it not more explicit to have them listed in the type like `| 'span' // you can use span here for <reason>` ?

That's not blocking :)